src/components/pages/Cases/CaseTable.js

```diff
@@ -38,7 +38,7 @@ export default function CaseTable(props) {
   };
 
   const { caseData, userInfo, savedCases, setSavedCases } = props;
-
+  // console.log('CASE DATA', caseData)
```

Comment:
Could this commented out console log be removed?

Reply:
Agreed - this should be removed before this PR gets Merged!